Hardcode the names and URLs of some additional members to scrape

mhl opened the pull request:
These members of the House of Representatives don't appear in searches
on the www.nass.gov.ng website, nor on the list of members on that site.
This commit adds manually their pages, which I found via Google searches
for the people who were reported as having gone missing in the
everypolitician-data pull request summarizer.

(n.b. There are still two from that list whom I can't find via Google.)

mhl committed:
Hardcode the names and URLs of some additional members to scrape

These members of the House of Representatives don't appear in searches
on the www.nass.gov.ng website, nor on the list of members on that site.
This commit adds manually their pages, which I found via Google searches
for the people who were reported as having gone missing in the
everypolitician-data pull request summarizer.

(n.b. There are still two from that list whom I can't find via Google.)

tmtmtmtm commented on scraper.rb:
```diff
@@ -115,6 +115,37 @@ def name_with_title
 
+# The search index and list pages seem to be missing some members of
+# the National Assembly, so add them manually to the pages to be scraped:
+members.concat [
```
`.concat` here will simply add these onto the end, even if they're already there (i.e. should the search start returning them again), which doesn't seem ideal. It's not a massive issue, but that would involve not only re-fetching, re-parsing, and re-saving the data, but also pushing it to the scraped page archive again, so it would be good to avoid that.

As you don't actually need the `name` part here, as the loop is only dealing with the URL, I'd be tempted to first build the list of URLs from the `SearchPage`, then `|` this second list with it, and then do the fetch loop over that.

mhl replied:
I thought about that and figured it wasn't a big deal. But I take your point - I've made the chnage you suggest in 271c031.

mhl committed:
Remove the possibility of scraping a member's URL multiple times

Previously, it would have been the case that if the Nigerian National
Assembly website was fixed and one of the people we added manually to
the list of URLs to be scraped reappeared in the search index, their
page would end up being scraped twice.

This commit uses @tmtmtmtm's suggestion to only collect URLs from the
search page results (since the name was unused anyway) and use the |
operator on the array of URLs to ensure we don't add URLs that are
already present.
